pywwise/ak/wwise/core/profiler.py

```diff
@@ -21,88 +35,503 @@ def __init__(self, client: _WaapiClient):
 		self.state_changed: _RefEvent
 		self.switch_changed: _RefEvent
 	
-	def enable_profiler_data(self):
+	def enable_profiler_data(self, data_types: set[tuple[EDataTypes, bool]] = None) -> bool:
```

Comment:
No need to have `None` as the default here. Make it a required param by removing the default.